doc/install/index.rst

```diff
@@ -1,72 +1,74 @@
-=======================
- Installation (Manual)
-=======================
+.. _install-overview:
 
+===============
+Installing Ceph
+===============
 
-Get Software
+There are different options to install Ceph. Each solution meet different
+requirements, like ease of maintenance.
+
+
+ceph-ansible
 ============
 
-There are several methods for getting Ceph software. The easiest and most common
-method is to `get packages`_ by adding repositories for use with package
-management tools such as the Advanced Package Tool (APT) or Yellowdog Updater,
-Modified (YUM). You may also retrieve pre-compiled packages from the Ceph
-repository. Finally, you can retrieve tarballs or clone the Ceph source code
-repository and build Ceph yourself.
+Installs Ceph using Ansible.
 
+* `docs.ceph.com/ceph-ansible <https://docs.ceph.com/ceph-ansible/>`_ 
 
-.. toctree::
-   :maxdepth: 1
+ceph-deploy
+===========
 
-	Get Packages <get-packages>
-	Get Tarballs <get-tarballs>
-	Clone Source <clone-source>
-	Build Ceph <build-ceph>
-    	Ceph Mirrors <mirrors>
-	Ceph Containers <containers>
+Install ceph using ceph-deploy
 
+* :ref:`ceph-deploy-index`
 
-Install Software
-================
 
-Once you have the Ceph software (or added repositories), installing the software
-is easy. To install packages on each :term:`Ceph Node` in your cluster. You may
-use  ``ceph-deploy`` to install Ceph for your storage cluster, or use package
-management tools. You should install Yum Priorities for RHEL/CentOS and other
-distributions that use Yum if you intend to install the Ceph Object Gateway or
-QEMU.
+cephadm
```

Comment:
Let's move this to the top?

Reply:
Then, how do you want to order the list of tools? Right now, this is alphabetical.

Reply:
maybe break into two sections... Recommended (with cephadm and rook) and then Other (with everything else, alphabetical?)